### ISSUE TITLE
Region rule results for related nodes are confusing

### ISSUE THREAD

**Issue by marcysutton** [dequelabs/axe-core#878]
From the web-a11y Slack:
> Running aXe on my webpage I am getting an error that the encompassing `html` tag must be in a landmark region. What regions have people put these under? I haven’t seen any examples online where the landmarks encompass the entire page. I have my `main` landmark nested within the `body`

>https://dequeuniversity.com/rules/axe/3.0/region?application=AxeChrome is the relevant error and the element its noting is `<html lang="en">` which is the root element

I've told folks repeatedly to "look at the related nodes", as the HTML tag is throwing them off. Users think the HTML tag itself is the problem, rather than the content contained within. I'm not sure what exactly we can do here, but given I've seen it come up more than once it seemed worth documenting in an issue.

This UX need is somewhat related to https://github.com/dequelabs/axe-core/issues/863 for skip links, but is more generic than that.

**Comment by WilcoFiers** [comment]
I have a proposal for this. Instead of having a single failure on the page, we fail all "outer" elements. That way we'll not end up with hundreds of violations. The way we do this is with an "after" method that takes what is now the related nodes, it filters out all of the nodes `contained` in other nodes, and sets those as violations. That should give us a nice short list.

**Comment by padmavemulapati** [comment]
Verified with content outside of a <main> landmark, then the violation no longer points to html element, working as expected.
![image](https://user-images.githubusercontent.com/47409579/74053178-4b7b0e80-4a01-11ea-9ad3-37099becf95b.png)
